homeassistant/components/iaqualink/sensor.py

```diff
@@ -53,7 +54,7 @@ def native_value(self) -> int | float | None:
         except ValueError:
             return float(self.dev.state)
 
```

Comment:
This one can be done in the constructor, needs a separate PR

Reply:
#100281